docs/tutorials/Remark.adoc

```diff
@@ -0,0 +1,347 @@
+    }
+
+    String remark = argMultimap.getValue(PREFIX_REMARK).orElse("");
+
+    return new RemarkCommand(index, remark);
+}
+----
+
+NOTE: Don't forget to update `AddressBookParser` to use our new `RemarkCommandParser`!
+
+If you are stuck, check out the sample link:https://github.com/j-lum/addressbook-level35/commit/c0f8419be4fb59bffafcd73026769459e5cf5c5e#diff-fc19ecee89c3732a62fbc8c840250508[here].
+
+== Adding to the model
+
+Now that we have all the information that we need, let's lay the groundwork for some _persistent_ changes.
+We acheive that by working with the `Person` model.
+Each field in a Person is implemented as a separate class (e.g. a Name object represents the person's name).
+That means we should add a Remark class so that we can use a Remark object to represent a remark given to a person.
+This protects them from illegal access and allows us to tightly couple logic specific to the field.
+
+=== Add a new `Remark` class
+
+Create a new `Remark` in `seedu.address.model.person`. Since a `Remark` is a field that is similar to `Address`, we can reuse a significant bit of code.
+
+A quick copy-paste and search-replace later, you should have something like link:https://github.com/j-lum/addressbook-level35/commit/c8e6048f0d81f2b4b7797e838226da823892129b#diff-af2f075d24dfcd333876f0fbce321f25[this].
+
+=== Make use of `Remark`
+
+Let's change `RemarkCommand` and `RemarkCommandParser` to use the new `Remark` class instead of plain `String`.
+These should be relatively simple changes.
+
+== Adding placeholder elements to the UI
+
+Without getting too deep into `fxml`, let's go on a 5 minute adventure to get some placeholder text to show up for each person.
+
+Simply add
+```
+@FXML
+private Label remark;
+```
+to link:https://github.com/j-lum/addressbook-level35/commit/06f9897d7088d22689de478c94612f8bcbff2964#diff-0c6b6abcfac8c205e075294f25e851fe[`PersonCard`].
+`@FXML` is an annotation that marks a private or protected field and makes it accessible to FXML.
+It might sound like Greek to you right now, don't worry -- we will get back to it later.
+
+Then insert
+
+```
+<Label fx:id="remark" styleClass="cell_small_label" text="\$remark" />
+```
+into link:https://github.com/j-lum/addressbook-level35/commit/06f9897d7088d22689de478c94612f8bcbff2964#diff-12580431f55d7880578aa4c16f249e71[`PersonListCard.fxml`].
+
+That's it! Fire up the application again and you should see something like this:
+
+.$remark shows up in each entry
+image::$Remark.png[]
+
+== Updating the model
+
+Since `PersonCard` displays data from a `Person`, we need to update `Person` to get our `Remark` displayed!
+
+=== Modify `Person`
+
+We change the constructor of `Person` to take a `Remark`.
+We will also need to define new fields accordingly to store our new addition.
+
+=== Update other usages of `Person`
+
+Unfortunately, a change to `Person` will cause other commands to break, let's fix these commands to use the updated `Person`!
```

Comment:
Make this statement clearer: are you going to explain how to do that in the sections below or the reader is supposed to do that by themselves.